source/service.go

```
@@ -69,16 +97,31 @@ func (sc *serviceSource) Endpoints() ([]*endpoint.Endpoint, error) {
 	return endpoints, nil
 }
 
-// endpointsFromService extracts the endpoints from a service object
-func endpointsFromService(svc *v1.Service) []*endpoint.Endpoint {
+func (sc *serviceSource) endpointsFromTemplate(svc *v1.Service) []*endpoint.Endpoint {
 	var endpoints []*endpoint.Endpoint
 
-	// Check controller annotation to see if we are responsible.
-	controller, exists := svc.Annotations[controllerAnnotationKey]
-	if exists && controller != controllerAnnotationValue {
-		return nil
+	var buf bytes.Buffer
+
+	if sc.fqdntemplate.Execute(&buf, svc) == nil { //TODO(ideahitme): if error is present skip or abort ?
```

as above please log or handle the error not silently drop it